layers/meta-balena-raspberrypi/recipes-kernel/picontrol/picontrol/0001-Use-modules_install-as-wanted-by-yocto.patch

```diff
@@ -15,14 +15,14 @@ index 56eab93..6d78b95 100644
 +++ b/Makefile
 @@ -44,6 +44,9 @@ EXTRA_CFLAGS += -D__KUNBUSPI_KERNEL__
  all: compiletime.h
- 	$(MAKE) ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- -C $(KDIR) M=$(PWD)  modules
- 
+ 	$(MAKE) ARCH=arm CROSS_COMPILE="$(CROSS_COMPILE)" -C $(KDIR) M=$(PWD)  modules
+
 +modules_install:
-+	$(MAKE) ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE) -C $(KDIR) M=$(shell pwd)  modules_install
```

Comment:
Is this patch really necessary? Do we get an error compiling the kernel if we don't change this patch? In my other PR with the kernel update I think these changes were not needed

Reply:
@floion It looks this is needed. It fails to compile when I disable remove this patch in the `picontrol.bb`. It was added here https://github.com/balena-os/balena-raspberrypi/commit/04d56a3b7ca3716b961e08b9f2b06876315eaf8c

The error it gives when I disable it is follows:
```
ERROR: Task (/home/ubuntu/balena-raspberrypi/build/../layers/meta-balena-raspberrypi/recipes-kernel/picontrol/picontrol.bb:do_install) failed with exit code '1'
```